src/test/java/edu/ksu/canvas/attendance/submitter/AssignmentValidatorUTest.java

```diff
@@ -86,46 +86,46 @@ public void setup() {
 
     @Test
     public void validateAttendanceAssignmentHappyPath() throws IOException, AttendanceAssignmentException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: java.io.IOException, edu.ksu.canvas.attendance.exception.AttendanceAssignmentException [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1160)